mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -20,19 +20,21 @@ pipeline {
         stage('Build MIOpenDriver') {
             steps {
                 dir('MIOpen') {
+                    sh 'mkdir install'
                     git branch: 'develop', poll: false,\
                                 url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
-                    cmake arguments: "-P install_deps.cmake --minimum",\
+                    cmake arguments: "-P install_deps.cmake --minimum --prefix ./MIOpen_deps",\
```

Comment:
[Question] What does this prefix do?

Reply:
It sets the install location. I figure the point is to avoid system-wide installs, which is a pretty reasonable thing to do, IMO.

Reply:
To specify the directory where MIOpen dependences are installed.

Reply:
If we are to install to `MIOpen_deps`, then why is the `mkdir install` command needed above?

Reply:
From @krzysz00:
> The deps are things like boost, but the make install puts the MIOpen in the right place.

Make sense.